.github/workflows/rust.yml

```diff
@@ -0,0 +1,47 @@
+name: Rust
+
+on:
+  push:
+    branches: [ master ]
+    paths:
+      - 'binding/rust/**'
+      - 'lib/linux/**'
+      - 'lib/mac/**'
+      - 'lib/raspberry-pi/**'
```

Comment:
should add jetson and beaglebone too right? our .NET SDK doesn't have jetson and beaglebone, so those weren't included